Configuration_adv.hpp

```diff
@@ -445,4 +447,26 @@
 // Note that if you use an app to control OAT, ANY debug output will likely confuse that app.
 // Debug output is useful if you are using Wifi to control the OAT or if you are issuing
 // manual commands via a terminal.
-//
+
+#if defined(OAT_DEBUG_BUILD)
+  // AVR based boards have numbers < 1000
+  #if BOARD < 1000
+    /*
+     * Debugging on the mega2560 using avr-stub dissallows application-code from
+     * using the normal (USB) serial port
+     */
+    // Disable debug output
+    #if defined(DEBUG_LEVEL)
+      #undef DEBUG_LEVEL
+    #endif
+    #define DEBUG_LEVEL (DEBUG_NONE)
+
+    // Disable serial control
+    #if defined(SUPPORT_SERIAL_CONTROL)
+      #undef SUPPORT_SERIAL_CONTROL
+    #endif
+    #define SUPPORT_SERIAL_CONTROL false
```

Comment:
We use 0 and 1 everywhere for defines, can we stick with that, please?

Reply:
Yeah I can change them to match the old defines, I just find it a lot more readable to use true booleans when you're looking through the logic

Reply:
I hear you. I think we decided on numbers because there's just always such confusion around using true, True or TRUE and unfortunately they are three distinct values to the preprocessor. The numbers are not ambiguous :-)

Reply:
Partially agree, but I just want to put it out there that `false` and `true` are defined in the standard as 0/1 respectively (["If the source type is `bool`, the value `false` is converted to `0` and the value `true` is converted to the value `1` of the destination type"](https://en.cppreference.com/w/cpp/language/implicit_conversion#Integral_conversions))

I'm gonna change it, but I'll always try and convince people for easier to read software that is strictly standards-compliant 😄